.github/workflows/php-unit-tests.yml

```diff
@@ -2,12 +2,28 @@ name: PHPUnit Tests
 
 on:
   push:
-    branches: [ trunk ]
+    branches:
+      - trunk
+      - develop
+    paths:
+      - "**.php"
+      - composer.json
+      - composer.lock
+      - .github/workflows/php-unit-tests.yml
   pull_request:
+    paths:
+      - "**.php"
+      - composer.json
+      - composer.lock
+      - .github/workflows/php-unit-tests.yml
 
 jobs:
   phpunit:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: [7.4, 8.0, 8.2, 8.3]
```

Comment:
I'm already seeing requests to add support for 8.4 to test matrices. Could we go ahead and add that here?

Reply:
Added in [23ff06c](https://github.com/woocommerce/snapchat-for-woocommerce/pull/10/commits/23ff06c49b476e2b3b3762231b09767ce6cc15c6)